ee/dataflatten/jsonl.go

```diff
@@ -13,11 +14,21 @@ func JsonlFile(file string, opts ...FlattenOpts) ([]Row, error) {
 		return nil, err
 	}
 	defer f.Close()
-	return Jsonl(f, opts...)
+
+	rawdata, err := io.ReadAll(f)
+	if err != nil {
+		return nil, fmt.Errorf("unable to read JSON: %w", err)
+	}
+
+	return flattenJsonl(rawdata, opts...)
+}
+
+func Jsonl(rawdata []byte, opts ...FlattenOpts) ([]Row, error) {
```

Comment:
Had to update this to make `Jsonl` signature compliant with `dataflatten.DataFunc` (`func(data []byte, opts ...FlattenOpts) ([]Row, error)`).

Reply:
I feel sad that past me didn't use io.Reader